### ISSUE TITLE
Release Twisted 16.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8231]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8231|
|Type|task|
|Created|2016-03-15 08:23:12Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.1.0-8231|

tasty tasty releases

<details><summary>Searchable metadata</summary>

```
trac-id__8231 8231
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_1 Twisted 16.1
branch__branches_release_16_1_0_8231 branches/release-16.1.0-8231
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1458030192884466 1458030192884466
changetime__1459790539583703 1459790539583703
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [47077]) Branching to release-16.1.0-8231.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Okay, all good. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

(the pyflakes and twistedchecker failures are spurious)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @hawkowl|
|-|-|

looks good. thanks!

I guess that when merging this you will have to fix the conflict, but should be fine.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47148]) Merge release-16.1.0-8231: Release Twisted 16.1

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8231](https://github.com/twisted/twisted/issues/8231)
